contracts/libs/Tx.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity ^0.5.15;
+
+library Tx {
+    // transaction:
+    // [sender<4>|receiver<4>|amount<4>]
+    uint256 public constant TX_LEN = 12;
+    uint256 public constant MASK_TX = 0xffffffffffffffffffffffff;
+    uint256 public constant MASK_STATE_INDEX = 0xffffffff;
+    uint256 public constant MASK_AMOUNT = 0xffffffff;
+    // positions in bytes
+    uint256 public constant POSITION_SENDER = 4;
+    uint256 public constant POSITION_RECEIVER = 8;
+    uint256 public constant POSITION_AMOUNT = 12;
+
+    function hasExcessData(bytes memory txs) internal pure returns (bool) {
```

Comment:
Can we have one function here that accepts (from, to, amount, signature) and emit bytes that I need to push to `submitBatch` function?

Reply:
The idea is that the node will call this function with the data and receive the `bytes calldata` that needs to be pushed on-chain.